smoke-test/src/test/java/org/opennms/smoketest/TopologyIT.java

```diff
@@ -114,6 +115,29 @@ public void removeFromFocus() {
             waitForTransition();
         }
 
+        public void expand() {
```

Comment:
Why do the expand() and collapse() methods catch `NoSuchElementException`? 
In addition the `waitForTransition()` only makes sense if the element was clicked.